tests/acl/test_acl.py

```diff
@@ -396,9 +396,12 @@ def setup(duthosts, ptfhost, rand_selected_dut, rand_unselected_dut, tbinfo, ptf
             # In multi-asic we need config both in host and namespace.
             if namespace:
                 acl_table_ports[''] += port
+    if len(port_channels) and topo in ["t0", "m0_vlan", "m0_l3"] or tbinfo["topo"]["name"] in ("t1-lag", "t1-64-lag",
+                                                                                               "t1-64-lag-clet",
```

Comment:
Why do we need to check `len(port_channels)` here?

Reply:
@bingwang-ms  if the there is no portchannels in the topo (such as t0-isoliatedu16d16s1), it should not enter this branch, it should go to the "else". if not the acl_table_ports will be empty.

Reply:
I see. Is that already covered by the check `topo in ["t0", "m0_vlan", "m0_l3"] or tbinfo["topo"]["name"] in ("t1-lag", "t1-64-lag",  "t1-64-lag-clet",` ?

Reply:
No, the t0-isoliatedu16d16s1 will still hit the condition  topo in ["t0", "m0_vlan", "m0_l3"] or tbinfo["topo"]["name"] in ("t1-lag", "t1-64-lag",  "t1-64-lag-clet",  so this is why I add the check for the len of port channel.